### ISSUE TITLE
Better pycairo installation instructions in docs

### ISSUE THREAD

**Issue by applemonkey496** [3b1b/manim#954]
I just wanted to point out that pycairo can be found in [this GitHub repo](https://github.com/pygobject/pycairo), and that there it says installing it is as simple as `pip install pycairo`. Am I missing something or should this information be included in the docs? It would be much easier than locating and installing the appropriate version from [here](https://www.lfd.uci.edu/~gohlke/pythonlibs/#pycairo) as the docs say. 

** Also, this is my first issue, so please tell me if I'm doing something wrong.

**Comment by einarf** [comment]
There is more to it:
https://pycairo.readthedocs.io/en/latest/getting_started.html

> Installing Pycairo requires pkg-config and cairo including its headers

The pre-compiled versions LFD is providing is probably the easiest option on windows.

**Comment by applemonkey496** [comment]
So does that mean that if I'm on MacOS (with Homebrew), to install pycairo, I would do `brew install cairo pkg-config` and then `pip3 install pycairo`?

**Comment by einarf** [comment]
Yeah they are. Could be good to PR doc changes in the pycairo project and link to that from Manim.
